html/feed/feed.js

```diff
@@ -0,0 +1,27 @@
+import logIn  from "./utils/helpers/loginFunc.js";
+
+
+async function fetchPosts (){
+    try {
+        const token = localStorage.getItem ('acces Token'); 
+        const getData ={
+            method:"GET",
+            credentials:"same-origin",
+            headers:{
+                "Content-Type":"application/json",
+            Authorization: 'Bearer ${token}',
+        },
+    };
+  
+const response = await fetch (apiUrl,getData); 
+console.log (response); 
+const json= await response.json();
+console.log (json); 
```

Comment:
the function needs to return something, if you wish to log, you can so before the return

Reply:
also remember to call the function  : )